app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,8 @@
 class ApplicationMailer < ActionMailer::Base
+
   default from: ENV['SHF_NOREPLY_EMAIL']
+
   layout 'mailer'
+
+
```

Comment:
Devise::Mailer inherits from this, as set in `initializers/devise.rb` (and then in the Devise::Mailer class: `Devise::Mailer < Devise.parent_mailer.constantize`

I *thought* that by having ShfMailer inherit from Devise::Mailer that Rails would use the views in `application/views/user_mailer`  but that's how it's happening.

So I'll change the hierarchy so that ShfMailer does inherit from ApplicationMailer.

Good catch.